src/screens/TopicMemberScreen/index.js

```diff
@@ -208,4 +207,4 @@ const TopicMemberScreen = () => {
     </SafeAreaProvider>
   );
 };
-export default withInteractionsManaged(TopicMemberScreen);
+export default TopicMemberScreen;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withInteractionsManaged` HOC has been removed from the export. If this was intentional and there are no interactions that need to be managed, then it's fine. However, if not, this could lead to unhandled interactions.
```diff
- export default TopicMemberScreen;
+ export default withInteractionsManaged(TopicMemberScreen);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->